2.6.md

```diff
@@ -2751,15 +2757,43 @@ Extended identifiers support in the OpenRTB specification allows buyers to use a
     <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
     <td><strong>Description</strong></td>
   </tr>
+<tr>
+    <td><code>inserter</code></td>
+    <td>string</td>
+    <td>The canonical domain name of the entity (publisher, publisher monetization company, SSP, Exchange, Header Wrapper, etc) that caused the ID array element to be added. This may be the operational domain of the system, if that is different from the parent corporate domain, to facilitate WHOIS and reverse IP lookups to establish clear ownership of the delegate system.<br>
+    </br>This should be the same value as used to identify sellers in an ads.txt file if one exists.<br>
```

Comment:
I'd rephrase this slightly, to look more like the following sentence. E.g. (assuming this is a true statement):

> For publishers, this would be the domain used to look up the ads.txt file for the request (if one exists).

Reply:
fixed per our conversation via email